src/character.cpp

```diff
@@ -596,6 +596,54 @@ std::vector<item_location> Character::nearby( const std::function<bool(const ite
     return res;
 }
 
+long int Character::i_add_to_container(const item &it)
```

Comment:
Do you actually use any `Character` methods here? I only see `inv`, which can be easily passed from above.
`Character` class is very bloated already.